### DISCUSSION TITLE
Is this project dying?

### DISCUSSION THREAD

**Discussion by HybridSolutions** [codex-team/editor.js#2381] [General]
Found this project recently and have to say it's very interesting and congrats to the team that pulled it off. Amazing!
It's the closest thing to WP block editor.

Sadly, it's an Open source only project so maintenance after a few years is going down the drain. The plugins architecture is interesting but is causing fundamental pieces of the machine to malfunction due to updates that don't show up (not blaming anyone).  Issues pile up and this os a red flag for me. What do you guys think? Do you feel the same or it's just my impression? 

Would love to work with it and even expand it but I'm not liking the way things are going. Maybe a paid version would save it since no one works for free on something forever and this project is a great candidate.

**Comment by neSpecc** [comment]
Thanks. 

Editor.js was created by the non-profit team [CodeX](http://twitter.com/codex_team), which consists of open-source development enthusiasts. Most of them have a main job and work on CodeX projects in their spare time. Nevertheless, the CodeX team is evolving, we have many ideas, and work is being done every day. Editor.js is our main project, and we continue making it the best player on the market.

Work on the Editor.js is divided into several directions: improving the current core, working on version 3.0, improving the ecosystem, and working with the community. Every few months, we [release](https://github.com/codex-team/editor.js/releases) a valued update. You can follow the progress on 3.0 [here](https://github.com/editor-js/document-model). This is one of the key modules of the future version.

Yes, we have accumulated many unresolved issues and pull requests. We are studying all of them, but often it takes significant time to provide a substantive response. Resources for this are currently limited, so, unfortunately, many questions remain unanswered. Nevertheless, each release includes bug fixes and several new features. Priority is given to the most critical bugs, popular requests, our own product and technical requirements, as well as requests from people who send donations.

To support our work and the development of Editor.js, you can subscribe to the donation at http://opencollective.com/editorjs. You can also join CodeX and participate in the development of the editor and its ecosystem.

**Reply by HybridSolutions** [reply]
First of all, I want to make clear that I understand that an open-source project has limited resources and depends on the time of each contributor. Maybe some more communication about the direction or even a public roadmap would fade this feeling that things are kind of stale when probably they are not. Hope the next version can bring more builtin features like text color, proper hyperlinks, text align among other "simple" things that now are only supported by plugins that are not supported anymore or with updates from 3 years ago. It's not good for the project to have that many outdated "dependencies". Will keep a close look into your progress and maybe even help you out, both in development and donations. Can't wait to see that 3.0 version ;)
